packages/aws-cdk/lib/init-templates/v1/sample-app/go/cdk.template.json

```diff
@@ -1,3 +1,6 @@
 {
-  "app": "go mod download && go run %name%.go"
+  "app": "go mod download && go run %name%.go",
+  "watch": {
+    "include": "**/*.go"
```

Comment:
Same comment for all of these and above as below, I don't want to repeat myself (for some reason, I went bottom to top on this review 😜).